Add object command (#290)
apache/kvrocks-website: docs/supported-commands.md

| COPY        | ✓                | v2.9.0        | Copies a key to a new key.                                                                                   |
| SORT        | ✓                | v2.9.0        | Sorts the elements in a list, set, or sorted set.                                                            |

### OBJECT subcommands

These commands are subcommands for `OBJECT`, using as `OBJECT DUMP` etc.

| SUBCOMMAND | Supported OR Not | Since Version | Description                                                                                                   |
| ---------- | ---------------- | ------------- | ------------------------------------------------------------------------------------------------------------- |

> | DUMP       | ✓                | v1.0.1        | Returns a serialized representation of the value stored at key.                                               |

Reviewer: ```suggestion
| DUMP       | ✓                | v1.0.1        | Dump the detailed information of the key.                                               |
```
